handbook/engineering/hiring/software-engineer-campaigns.md

```diff
@@ -0,0 +1,75 @@
+- Experience in at least one server side language, such as Go, Java, Python, PHP, or Ruby.
+- Experience building and testing end-to-end applications (e.g. unit, integration, and e2e testing).
+- Good knowledge of Git in particular and other version control systems in general.
+- Experience in SQL, Postgres or other transactional, relational databases. Able to model and design schemas.
+- Have worked with containers in production environments.
+
+## Not sure if this is you?
+
+We want a diverse, global team, with a broad range of experience and perspectives. If this job sounds great, but you’re not sure if you qualify, apply anyway! We carefully consider every application, and will either move forward with you, find another team that might be a better fit, keep in touch for future opportunities, or thank you for your time.
+
+## Learn more about us
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](https://about.sourcegraph.com/company/remote) that is [distributed across the world](https://about.sourcegraph.com/company/team). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](https://about.sourcegraph.com/handbook/people-ops/compensation) and [practical benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](https://about.sourcegraph.com/handbook).
+
+## Interview process
+
+<!-- 1. You [apply here](#todo). --><!-- This is commented out because I will be sending some outbound messages before accepting applications. -->
+
+1. The **Hiring Manager** reviews all the information you provided on your application to determine if you meet our qualifications for this role (if there is another open role we think you would be better qualified for, we will let you know).
+1. A **Recruiter** sets up a 30 minute call to learn more about what you are looking for, tell you about Sourcegraph, and answer any questions that you have.
+1. We set up a 1 hour interview with the **Hiring Manager**, who will tell you more about the role in the team. They will ask you about your past work and accomplishments and assess your suitability for the role.
+1. You complete a 3 hour [coding exercise](software-engineer-coding-exercise.md#frontend-coding-exercise) that we designed to test your technical ability and the responsibilities of the role as [listed above](#your-responsibilities).
+   - _Note: while we do not disqualify candidates who do not have React and TypeScript experience from taking the exercise, you will be required to use these for the exercise._
+1. An **Engineer** and a **Designer** will review and grade your submission. We will send you our review along with some follow-up questions.
```

Comment:
I _think_ we usually don't give reviews back to candidates. Could have a bad signal if we said "Well John didn't like (1), (2) and (3)" "Congrats here's your offer" - then there will always be this tension between John and the candidate. Not sure this this is a real concern though.

Reply:
I always sanitize these feedback, so it's coming from me. And the feedback _always_ includes both pros and cons. I'm happy to talk more in person about the process I have used in the past and the signal I get from it.

Reply:
As long as it's sanitized and anonymized, I think that's fine.

Reply:
@felixfbecker what is your view on doing the follow-up questions async vs in-person? Can we get the same signal from an async response that you normally look for in the sync call with a candidate?